db_migration/db_migration.sh

```diff
@@ -1,30 +1,35 @@
 # This code calls db migration.
 
+# To use the db migration, a klaytn client should not be running.
+# You need to have access to the DB.
+# To checkout migration status, `tail -f logs-body.out`
+
 # BIN file
-KLAYTN_BIN=~/klaytn/bin
+KLAYTN_BIN=~/klaytn/bin/ken
```

Comment:
Can I ask the reason why you change the path of BIN file?

Reply:
There could be a case when `kpn` or `kcn` is used instead of `ken`. 
* Before this pr, the binary file name (`ken`) was not included.
* After this pr, the binary file name (`ken`) is included.